packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -704,13 +705,15 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
             id={`${index}-line-${this._uniqueIdForGraph}`}
             d={line(singleStackedData)!}
             fill={'transparent'}
-            strokeWidth={3}
+            strokeWidth={points[index].lineOptions?.strokeWidth ?? 3}
             stroke={this._colors[index]}
             opacity={this._getLineOpacity(points[index]!.legend)}
             onMouseMove={this._onRectMouseMove}
             onMouseOut={this._onRectMouseOut}
             onMouseOver={this._onRectMouseMove}
-            {...points[index]!.lineOptions}
+            strokeDasharray={points[index].lineOptions?.strokeDasharray}
+            strokeDashoffset={points[index].lineOptions?.strokeDashoffset}
+            strokeLinecap={points[index].lineOptions?.strokeLinecap ?? 'round'}
```

Comment:
Is 'round' the current default behavior.

Reply:
It is the default value in LineChart but not in AreaChart. Removed it.